source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -56,6 +60,28 @@ The following configurations, which are explained in the [IPAM types](#ipam-type
 
 * static addresses
 * stateful DHCPv6
+* stateless DHCPv6
+
+### Stateless DHCPv6 / SLAAC motivation
+
+ISC DHCP client - dhclient - has up to now relied on an hardcoded prefix length
+of 64 bits. That was in line with [RFC 5375](https://tools.ietf.org/html/rfc5375#section-3) concept of Global Unique Addresses and Unique Local Addresses.
+
+Unfortunately, the use of an hardcoded prefix length will lead to clients
+believing they are on-link, when in fact they are not, as discussed in
+[RFC 5942](https://tools.ietf.org/html/rfc5942). This would happen for any
+network whose prefix length is different than 64 bits.
+
+[This](https://kb.isc.org/docs/aa-01141) ISC's article clearly states that a
+properly functioning system should use a prefix length of 128, and get any
+on-link information from the RA messages sent by routers.
+
+Ubuntu [bug](https://bugs.launchpad.net/ubuntu/+source/isc-dhcp/+bug/1609898) tracks that very same issue, and that behavior is now found
+in EL8 systems - e.g. dhclient default to a /128 network prefix, and relies on
+RA messages to properly configure it.
+
+Thus, RA messages support is required to properly configure the guest IPv6
+addresses, and bug [1685983](https://bugzilla.redhat.com/show_bug.cgi?id=1685983) was opened to track its support in ovirt-provider-ovn.
 
```

Comment:
The subsection `### Stateless DHCPv6` is for a technical guy like me very interesting, but for other people maybe not straight to understand.
Maybe you could condense the message on a high abstraction level? From my understanding, it is that the network prefix lenght and maybe default gateway is missing, which leads to an unsuable (?) config for RHEL8 guests.

Reply:
Agree

Reply:
I can probably re-phrase this somehow, but, I think that 'why' it stopped working in EL8 is important, and probably even more important, 'why' it was changed in EL8. 

Added a TL;DR section for anyone who doesn't care about that.